lib/server.ts

```diff
@@ -3,6 +3,7 @@ import * as express from 'express';
 import * as http from 'http';
 
 import { Stream } from 'stream';
+import * as fs from 'fs';
```

Comment:
```suggestion
import * as fs from 'fs/promises';
```
https://nodejs.org/docs/latest-v14.x/api/fs.html#fs_fspromises_access_path_mode